fables/api.py

```diff
@@ -8,8 +8,9 @@
 from typing import Dict, IO, Iterable, Optional, Tuple, Union
 
 from fables.constants import MAX_FILE_SIZE
+from fables.parse import ParseVisitor
+from fables.results import ParseResult
 from fables.tree import FileNode, node_from_file
-from fables.parse import ParseResult, ParseVisitor
```

Comment:
This was a bad import, but it didn't materialize as a runtime error because it is only used as a type annotation.